static/doc.json

```diff
@@ -1086,6 +1086,104 @@
           }
         }
       }
+    },
+    "/nh/events": {
+      "get": {
+        "summary": "All New Horizons events",
+        "description": "Get a list of all ongoing or upcoming events in *Animal Crossing: New Horizons*.",
+        "parameters": [
+          {
+            "in": "header",
+            "name": "X-API-KEY",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "format": "uuid"
+            },
+            "description": "Your UUID secret key, granted to you by the Nookipedia team. Required for accessing the API."
+          },
+          {
+            "in": "header",
+            "name": "Accept-Version",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The version of the API you are calling, written as `1.0.0`. This is specified as required as good practice, but it is not actually enforced by the API. If you do not specify a version, you will be served the latest version, which may eventually result in breaking changes."
+          },
+          {
+            "in": "query",
+            "name": "eventyear",
```

Comment:
You can remove "event" from all these query names, since it is implied by the endpoint name.